### ISSUE TITLE
Memory leaks in lib

### ISSUE THREAD

**Issue by GoogleCodeExporter** [niltsh/panoramagl#3]
```
What steps will reproduce the problem?
1. Start a pano and turn around 10 minutes fast
2. Memory Warnung Level 1 and 2
3. Build and Analyse showed up 16 leaks
4. Instruments found a leak in PLSphere init

What is the expected output? What do you see instead?
Crashed

What version of the product are you using? On what operating system?
iPad 3.2

Please provide any additional information below.
Testes on iPad with a Simple splitview

```

Original issue reported on code.google.com by `sd8...@googlemail.com` on 11 Aug 2010 at 5:53

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```

I found a small workaround:

The leaks come from PurplePushEvent (from the Core) and are only on 3.x 
devices, not on 4.x devices :( The leaks are in the original Hellopanorama 
Example too.

If you deactivate the accelerometer between touchesbegan: and touchesended: it 
will not leak. I think it´s a Core OS problem.

The PLSphere leak is a other problem.
```

Original comment by `sd8...@googlemail.com` on 16 Aug 2010 at 12:31

**Comment by GoogleCodeExporter** [comment]
```
by the way: I found this solution here: 
http://www.iphonedevsdk.com/forum/iphone-sdk-development/33754-strange-leaks-fou
nded.html#post206374
```

Original comment by `sd8...@googlemail.com` on 16 Aug 2010 at 12:37

**Comment by GoogleCodeExporter** [comment]
```
Memory leaks were fixed in 0.1 version, please check last version
```

Original comment by `javbae...@gmail.com` on 5 Jul 2012 at 3:58
- Changed state: **Fixed**
